ci/ReleaseTool/PrepCommand.cs

```diff
@@ -5,7 +5,7 @@ namespace ReleaseTool
 {
     /// <summary>
     ///     Runs the steps required to cut release candidate branches in all repos:
-    ///     UnrealGDK, UnrealGDKExampleProject, UnrealEngine, UnrealGDKEngineNetTest, UnrealGDKTestGyms and TestGymBuildKite.
+    ///     UnrealGDK, UnrealGDKExampleProject, UnrealEngine, UnrealGDKEngineNetTest, UnrealGDKTestGyms, TestGymBuildKite and UnrealGDKBuild.
```

Comment:
```suggestion
    ///     UnrealGDK, UnrealGDKExampleProject, UnrealEngine, UnrealGDKEngineNetTest, UnrealGDKTestGyms, TestGymBuildKite, and UnrealGDKBuild.
```
Extremely important Oxford comma.